app/src/main/java/openfoodfacts/github/scrachx/openfood/features/splash/SplashActivity.kt

```diff
@@ -2,8 +2,11 @@ package openfoodfacts.github.scrachx.openfood.features.splash
 
 import android.annotation.SuppressLint
 import android.content.pm.ActivityInfo
+import android.os.Build
 import android.os.Bundle
 import androidx.activity.viewModels
+import androidx.core.splashscreen.SplashScreen
+import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
```

Comment:
I would remove this import and use SplashScreen.installSplashScreen(), as we use this function just one time

Reply:
Since `installSplashScreen()` is an extension function on the `Activity` class, I'm not sure it's possible to exclude this import statement. See https://kotlinlang.org/docs/extensions.html#scope-of-extensions
Take a look at `installSplashScreen()`'s signature (code from the `androidx.core.splashscreen` package):
```kotlin
public fun Activity.installSplashScreen(): SplashScreen {
    //
}
```
I might be wrong, however, if so, please tell me how could I resolve this issue. Thanks!

Reply:
You're right. I reviewed the code in github so no different colors for extensions! Thank yoU!